diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+Users/grace/wdi/try
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+an online database where users can upload files/projects for others to see, use, and edit using git, a version control program.
 
```

Comment:
Great answer but it's mostly the UI. Github is not Git, and it's not version control. It's the web client UI for using git.